packages/compass-aggregations/src/components/pipeline-explain/explain-indexes.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Badge, BadgeVariant, Body } from '@mongodb-js/compass-components';
+import type { IndexInformation } from '@mongodb-js/explain-plan-helper';
+
+type ExplainIndexesProps = {
+  indexes: IndexInformation[];
+};
+
+export const ExplainIndexes: React.FunctionComponent<ExplainIndexesProps> = ({
+  indexes,
+}) => {
+  if (indexes.filter(({ index }) => index).length === 0) {
+    return <Body weight="medium">No index available for this query.</Body>;
+  }
+
+  return (
+    <div>
+      {indexes.map((info, idx) => (
+        <Badge key={idx} variant={BadgeVariant.LightGray}>
+          {info.index} {info.shard && <>({info.shard})</>}
```

Comment:
React noob question – what’s difference between `{info.shard && <>({info.shard})</>}` and just `{info.shard}`?

Reply:
Goal here is to show the index along with its shard name, if defined.
In case `info.shard` is not defined, the former does not render empty parenthesis, while the latter does.

Just to avoid something like this: `host_id_idx ()`

Reply:
Well I did miss those parentheses :) All good then!

Reply:
FWIW making it a template literal will make it a bit more readable in this case otherwise it's really easy to miss these parentheses

```
`(${info.shard})`
```